tests/unit/tasks/test_tool_task.py

```diff
@@ -10,6 +10,133 @@
 
 
 class TestToolTask:
+    TARGET_TOOLS_SCHEMA = {
```

Comment:
Originally added this to validate the existence of `minItems` in the schema, but now it is being added during prompt rendering. Can remove these tests, but I think there is value in having them.